Makefile

```diff
@@ -66,3 +66,12 @@ lint-autocorrect:
 
 lint:
 	(swiftlint lint Sources SampleApps/AEPCommerceDemoApp)
+
+check-version:
+	(sh ./script/version.sh $(VERSION))
```

Comment:
should be uppercase Script `./Script/version.sh`
same for the other checks below